src/components/table/LogTableHeader.vue

```diff
@@ -4,19 +4,19 @@
 -->
 <template>
 	<th>
-		<div class="log-table__header-wrapper">
+		<div class="log-table-header__wrapper">
 			<NcButton v-if="sortable"
-				type="tertiary-no-background"
+				:type="props.sorted ? 'primary' : 'tertiary-no-background'"
```

Comment:
did you try the secondary already? I think that would be less prominent but enought to highlight it :)

Reply:
```suggestion
				:type="props.sorted ? 'secondary' : 'tertiary-no-background'"
```

Reply:
We can do this, but discussed on the nextcloud-vue library a pressed button should have the `primary` styling, so not sure about this.

(I personally find the secondary better for this purpose): 
![image](https://github.com/nextcloud/logreader/assets/1855448/29412449-f423-4ce4-9ca7-b581df026cae)

Reply:
Yes I would also use the secondary button here.